pennylane_lightning/lightning_kokkos/lightning_kokkos.toml

```diff
@@ -58,7 +58,8 @@ StatePrep              = {}
 # Gates which should be translated to QubitUnitary
 [operators.gates.matrix]
 
-BlockEncode            = {}
+BlockEncode            = {properties = [ "controllable" ]}
+'C(GlobalPhase)'       = {}
```

Comment:
As far as I'm aware global phase is not controllable at the C++ layer. The `C(...)` string is the wrong format anyway no?